src/utils/const.ts

```diff
@@ -94,39 +94,39 @@ export const ContractHashTags: ContractHashTag[] = [
   {
     codeHash: '0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8',
     txHashes: [
-      '0x6495cede8d500e4309218ae50bbcadb8f722f24cc7572dd2274f5876cb603e4e-0',
       '0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c-0',
+      '0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37-0',
     ],
     tag: 'secp256k1_blake160',
     category: 'lock',
   },
   {
     codeHash: '0x5c5069eb0857efc65e1bca0c07df34c31663b3622fd3876c876320fc9634e2a8',
     txHashes: [
-      '0x6495cede8d500e4309218ae50bbcadb8f722f24cc7572dd2274f5876cb603e4e-1',
       '0x71a7ba8fc96349fea0ed3a5c47992e3b4084b031a42264a018e0072e8172e46c-1',
+      '0xf8de3bb47d055cdf460d93a2a6e1b05f7432f9777c8c474abf4eec1d4aee5d37-1',
     ],
     tag: 'secp256k1 / multisig / locktime',
     category: 'lock',
   },
   {
-    codeHash: '0x6a3982f9d018be7e7228f9e0b765f28ceff6d36e634490856d2b186acf78e79b',
-    txHashes: ['0x69c70d65832cdfd97fe78d32eb25f840232f6b8cb6445464f11dad891b11fd83-0'],
+    codeHash: '0x86a1c6987a4acbe1a887cca4c9dd2ac9fcb07405bbeda51b861b18bbf7492c4b',
```

Comment:
What if mainnet and testnet have different values?

Reply:
`txHashes` is an array which includes mainnet tx and testnet tx and `codeHash` is same in mainnet and testnet now.

If `codeHash` has different values in mainnet and testnet later, I will refactor its data type.

Reply:
`codeHash` will definitely be different.

Reply:
Why? `codeHash` now is the same.

Reply:
I mean anyone-can-pay and sudt not secp256k1.